.gitignore

```
@@ -5,6 +5,8 @@ node_modules
 npm-debug.log
 coverage
 module.config.js
+env.config.js
+env.config.jsx
```

nit: might be able to tackle both of these in 1 line?

```suggestion
env.config.js*
```

---

I'm learning something new every day haha. Thanks for the suggestion!